WordPress/gutenberg
# Remove additional Typeset screen and surface typesets in the typography panel

## What?

This PR surfaces the typography variations ("typesets") directly in the Typography panel, removing the need for a separate Typeset screen right now.

## Why?


See https://github.com/WordPress/gutenberg/issues/65575

## How?


## Testing Instructions
1. Activate a theme with typography variations, like TT5
2. Visit the site editor > global styles > typography
3. Verify you can see the typesets and switch them





### Testing Instructions for Keyboard


## Screenshots or screencast 

Trunk | This PR
--- | ---
![Screenshot 2024-09-23 at 2 30 28 PM](https://github.com/user-attachments/assets/3e86cd98-6c8e-4efe-98a5-8f18629c3b97) | ![Screenshot 2024-09-23 at 2 25 30 PM](https://github.com/user-attachments/assets/73ed508d-64ab-4686-850b-f04d47601f7e)
